src/analyzer/plugins/analyzerkeyfinder.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <keyfinder/keyfinder.h>
+
+#include <QObject>
+#include <vector>
+
+#include "analyzer/plugins/analyzerplugin.h"
+#include "util/memory.h"
```

Comment:
You can now include `<memory>` directly, we should deprecate or remove #include "util/memory.h"